daemon/container.go

```diff
@@ -237,6 +237,21 @@ func (daemon *Daemon) verifyContainerSettings(hostConfig *containertypes.HostCon
 				return nil, err
 			}
 		}
+
+		// Validate the healthcheck params of Config
+		if config.Healthcheck != nil {
+			if config.Healthcheck.Interval < 0 {
```

Comment:
I think this should validate less than `1s`, wdyt as I don't think we can guarantee sub-second checking.